File: server-scala-play-server/app/model/ProfileShortResponse.scala
package model

import play.api.libs.json._

/**
  * Represents the Swagger definition for ProfileShortResponse.
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"), date = "2026-02-09T20:57:39.376804970Z[Etc/UTC]", comments = "Generator version: 7.20.0-SNAPSHOT")
case class ProfileShortResponse(
  valid: Option[Boolean],
  message: Option[String],
  version: Option[Double],
  serializeNulls: Option[Boolean],
  startTimeLog: Option[Long],
  errorCode: Option[String],
  request: Option[List[NameStringValueResponse]],
  accountId: Option[Long],
  locationDisplay: Option[String],
  display: Option[String],
  username: Option[String],
  profileImage: Option[String],
  profileAvatar: Option[AssetShortResponse],
  returning: Option[String]
)

object ProfileShortResponse {
  implicit lazy val profileShortResponseJsonFormat: Format[ProfileShortResponse] = Json.format[ProfileShortResponse]
}

File: server-aspnet-fastendpoints/src/Org.OpenAPITools/Models/ProfileShortResponse.cs
namespace Org.OpenAPITools.Models;


/// <summary>
/// 
/// </summary>
public class ProfileShortResponse 
{
    public bool Valid { get; set; }
    public string Message { get; set; }
    public double VarVersion { get; set; }
    public bool SerializeNulls { get; set; }
    public long StartTimeLog { get; set; }
    public string ErrorCode { get; set; }
    public List<NameStringValueResponse> Request { get; set; }
    public long AccountId { get; set; }
    public string LocationDisplay { get; set; }
    public string Display { get; set; }
    public string Username { get; set; }
    public string ProfileImage { get; set; }
    public AssetShortResponse ProfileAvatar { get; set; }
    public string Returning { get; set; }
}


